ks3651.md

```diff
@@ -0,0 +1,31 @@
+Open Source Development
+
+Name: KeunWoo Song
+
+UNI: ks3651
+
+Github: https://github.com/kw9212/project-proposals-s2023/tree/keunwoo
+
+## Project Proposal
+
+As of now, I would like to contribute on one of the four open source projects below:
```

Comment:
these should be in the `Project Contributions` section, and in this section you should brainstorm the type of project you wish to create

Reply:
Thank you for pointing this out, professor. I changed this too.

As a beginner with Git and GitHub, during my experimentation, I accidentally revised the contents of HW0(keunwoo branch) to what should have been assigned to HW1. I created separate markdown files for each project and contribution plan and submitted a pull request for HW1.